spec/factories/api_tokens.rb

```diff
@@ -3,7 +3,7 @@
 
   factory :api_token do
     access_token
-    step
+    step { create(:approval_step) }
```

Comment:
yes and no: I had to change this because previously the `step` factory was an approval step, which was what lead to the failure in the `root_step` spec I created (root step needs to be parallel or serial). So  I changed the `step` spec definition, which required me to change this because api_token specs failed when this factory used a vanilla step rather than an approval or purchase step (api tokens can be related to either)